### DISCUSSION TITLE
Rota para rastreio de encomendas

### DISCUSSION THREAD

**Discussion by gabrielboliveira** [BrasilAPI/BrasilAPI#246] [Ideas]
Olá! Parabéns pela iniciativa, excelente projeto!

Creio ser relevante uma rota para obter as informações dos [rastreios de encomendas](https://www2.correios.com.br/sistemas/rastreamento/). A [API SRO](https://www.correios.com.br/enviar-e-receber/precisa-de-ajuda/manual_rastreamentoobjetosws.pdf) segue uma estrutura semelhante a de busca por CEP, limitando acesso CORS. 

Poderia utilizar esse pacote: [tracking-correios](https://github.com/gabrielboliveira/tracking-correios). Várias issues foram criadas com o mesmo assunto: limitação de CORS para utilizar pelo navegador.

Cache longo talvez não seja tão relevante nesse caso específico, pois os dados alteram com uma frequência bem maior que CEPs.

Acreditam ser relevante para esse projeto?
Valeu!

**Comment by atiladelcanton** [comment]
Olhei seu pacote, e acho que faz relevancia com o projeto ao meu ver.

Por sinal prabéns pelo pacote de rastreio.

**Comment by filipedeschamps** [comment]
Sensacional @gabrielboliveira e com certeza é relevante! E parabéns pelo `tracking-correios` e muito obrigado pela citação lá no final 👍 

O que acha de aguardarmos a resolução dos PRs sobre testes automatizados e ci/cd (https://github.com/filipedeschamps/BrasilAPI/pull/11 https://github.com/filipedeschamps/BrasilAPI/pull/27 https://github.com/filipedeschamps/BrasilAPI/pull/29) para dai iniciarmos a programação desse novo endpoint? Seria uma honra ter a sua participação, principalmente por ter mais conhecimento nesse serviço 👍

**Comment by gabrielboliveira** [comment]
@filipedeschamps Excelente! Acho uma ótima ideia. Estou a disposição para ajudar quando necessário.

**Comment by pedrosancao** [comment]
o PR #151 precisa de trabalho, adicionei uma revisão de código mas não tenho acesso para adicionar commits no mesmo PR
